uploadNeuroDB/NeuroDB/ImagingUpload.pm

```diff
@@ -187,11 +187,11 @@ sub IsCandidateInfoValid {
         #1) Check to see if the file is of type DICOM###########
         #2) Check to see if the header matches the patient-name#
         ########################################################
-        if ( ( $_ ne '.' ) && ( $_ ne '..' ) ) {
+        if ( ( $_ ne '.' ) && ( $_ ne '..' ) && (basename($_) ne '.DS_Store')) {
```

Comment:
I would also like to see && (basename($_) !~ '^\._')

This would exclude all files starting with dot underscore.